bin/wasabi.sh

```diff
@@ -247,7 +249,7 @@ status() {
 }
 
 package() {
-  profile=build
+  [ "${profile}" == "${profile_default}" ] && profile=build
```

Comment:
yeah that is a ramification.

this is needed to get build travis correctly (signing jars, etc) ... we could look at melding the profiles (build -> development) ... but we'd need to look into the ramifications, reasons it was separated out, etc.

honestly, i would like to revisit this ... but not block on this one at this point in time. as such, added a FIXME